test/controllers/HomeControllerSpec.scala

```diff
@@ -21,7 +21,7 @@ class HomeControllerSpec extends FrontEndTestHelper {
       status(home) mustBe OK
```

Comment:
Appear to be a couple of unused imports:

- `import play.api.mvc.Result`
- `import scala.concurrent._`